docs/src/whatsnew/latest.rst

```diff
@@ -35,10 +35,11 @@ This document explains the changes made to Iris for this release
    non-existing paths, and added expansion functionality to :func:`~iris.io.save`.
    (:issue:`4772`, :pull:`4913`)
 
-#. `@hsteptoe`_ and `@trexfeathers`_ (reviewer) added :func:`iris.pandas.as_data_frame`,
-   which provides improved conversion of :class:`~iris.cube.Cube`\s to 
-   :class:`~pandas.DataFrame`\s. This includes better handling of multiple 
-   :class:`~iris.cube.Cube` dimensions, auxiliary coordinates and attribute information.
+#. `@hsteptoe`_ and `@trexfeathers`_ improved
+   :func:`iris.pandas.as_data_frame`\'s conversion of :class:`~iris.cube.Cube`\s to
+   :class:`~pandas.DataFrame`\s. This includes better handling of multiple
+   :class:`~iris.cube.Cube` dimensions, auxiliary coordinates and attribute
+   information. **Note:** the improvements are opt-in, via :class:`iris.Future`.
```

Comment:
Perhaps mentions the full future flag name `iris.FUTURE.pandas_ndim` so a user doesn't have to go looking in the docs for it (like how we do [here in the 3.3 release](https://github.com/SciTools/iris/blame/eb2144830bc50618090212f90f66b55cb6914b18/docs/src/whatsnew/3.3.rst#L20))

Reply:
3b9f21afce352e480454393676e2f92b05712972